functions/consumer/rabbit-consumer/src/main/java/org/springframework/cloud/fn/consumer/rabbit/RabbitConsumerConfiguration.java

```diff
@@ -144,25 +146,49 @@ public void destroy() {
-//						credentialsRefreshService, connectionNameStrategy, connectionFactoryCustomizers);
-				cachingConnectionFactory.setConnectionNameStrategy(
-						connectionFactory -> "rabbit.sink.own.connection");
-				cachingConnectionFactory.afterPropertiesSet();
-				return cachingConnectionFactory;
-			}
-
-		}
+	private ConnectionFactory buildLocalConnectionFactory() throws Exception {
+		this.ownConnectionFactory = rabbitConnectionFactory(this.bootProperties, this.resourceLoader,
+				this.credentialsProvider, this.credentialsRefreshService, this.connectionFactoryCustomizers);
+		return this.ownConnectionFactory;
 	}
 
-}
-
-
+	/**
+	 *
+	 * @param properties
+	 * @param resourceLoader
+	 * @param credentialsProvider
+	 * @param credentialsRefreshService
+	 * @param connectionFactoryCustomizers
+	 * @return
+	 * @throws Exception
+	 */
+	private CachingConnectionFactory rabbitConnectionFactory(RabbitProperties properties, ResourceLoader resourceLoader,
+			ObjectProvider<CredentialsProvider> credentialsProvider,
+			ObjectProvider<CredentialsRefreshService> credentialsRefreshService,
+			ObjectProvider<ConnectionFactoryCustomizer> connectionFactoryCustomizers) throws Exception {
+
+		/* NOTE: This is based on RabbitAutoConfiguration.RabbitConnectionFactoryCreator
+		 * 		https://github.com/spring-projects/spring-boot/blob/c820ad01a108d419d8548265b8a34ed7c5591f7c/spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/amqp/RabbitAutoConfiguration.java#L95
+		 * [UPGRADE_CONSIDERATION] this should stay somewhat in sync w/ the functionality provided by its original source.
+		 */
+		RabbitConnectionFactoryBean connectionFactoryBean = new RabbitConnectionFactoryBean();
+		RabbitConnectionFactoryBeanConfigurer connectionFactoryBeanConfigurer = new RabbitConnectionFactoryBeanConfigurer(resourceLoader, properties);
+		connectionFactoryBeanConfigurer.setCredentialsProvider(credentialsProvider.getIfUnique());
+		connectionFactoryBeanConfigurer.setCredentialsRefreshService(credentialsRefreshService.getIfUnique());
+		connectionFactoryBeanConfigurer.configure(connectionFactoryBean);
+		connectionFactoryBean.afterPropertiesSet();
+
+		com.rabbitmq.client.ConnectionFactory connectionFactory = connectionFactoryBean.getObject();
+		connectionFactoryCustomizers.orderedStream()
+				.forEach((customizer) -> customizer.customize(connectionFactory));
+
+		CachingConnectionFactory cachingConnectionFactory = new CachingConnectionFactory(connectionFactory);
+		CachingConnectionFactoryConfigurer cachingConnectionFactoryConfigurer = new CachingConnectionFactoryConfigurer(properties);
+		cachingConnectionFactoryConfigurer.setConnectionNameStrategy(cf -> "rabbit.sink.own.connection");
+		cachingConnectionFactoryConfigurer.configure(cachingConnectionFactory);
+		cachingConnectionFactory.afterPropertiesSet();
+
+		return cachingConnectionFactory;
+	}
 
+}
```

Comment:
New line in the end of each file.
But still: I guess the formatter should do that for us.
Only need to learn how to use it 😄

Reply:
I agree. Is javaformat plugin installed in this repo? Or are you saying we should add it?

Reply:
No, I only see `spring-javaformat-checkstyle` which does don't do reformatting: https://github.com/spring-io/spring-javaformat#source-formatting
Sorry for a noise.

We may reconsider this in a different PR

Reply:
I reformatted w/in IDEA (using the spring cloud build formatter) and it did correct the finally. I am about to submit another commit w/ that as well as a checkstyle fix.